packages/data-point-express/lib/middleware.js

```diff
@@ -42,20 +42,23 @@ function createErrorMessage(err) {
   };
 }
 
-function resolveReducer(dataPoint, reducer, options, res, initialValue = {}) {
-  dataPoint
-    .transform(reducer, initialValue, options)
-    .then(acc => {
-      if (typeof acc.value === "string") {
-        res.send(acc.value);
-        return;
-      }
-      res.json(acc.value);
-    })
-    .catch(err => {
-      res.status(400).send(createErrorMessage(err));
-    })
-    .done();
+async function resolveReducer(
+  dataPoint,
+  reducer,
+  options,
+  res,
+  initialValue = {}
+) {
+  try {
+    const acc = await dataPoint.transform(reducer, initialValue, options);
+    if (typeof acc.value === "string") {
```

Comment:
Do we have lodash here? You could use _.isString

Reply:
we are moving away from lodash as much as possible, especially on things where vanilla JS suffices, the reasons are around performance + size of data-point itself when required.

in this specific case, this change would be out of the scope of this PR which is to move to async/await